tests/acceptance/addon-smoke-test-slow.js

```diff
@@ -164,6 +164,16 @@ describe('Acceptance: addon-smoke-test', function() {
       });
   });
 
+  it('ember addon with linting errors', function() {
+    return copyFixtureFiles('addon/with-linting-errors')
+      .then(function() {
+        return ember(['test']);
+      })
+      .then(function(result) {
```

Comment:
Huh, interesting.  After working with this a little, it seems like the linting is actually passing, even with that `with()` statement in the file, which I would have thought should make the linter unhappy.  Need to investigate this a bit more.